src/views/Room/RollOptionsMenu/index.jsx

```
@@ -0,0 +1,66 @@
+import { useRef, useState } from 'react';
+
+const RollOptionsMenu = ({ options, selectedRoll, handleMenuItemClick }) => {
+  const [open, setOpen] = useState(false);
+  const anchorRef = useRef(null);
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClose = (event) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)